FirebaseSessions/Sources/FirebaseSessionsError.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Contains the list of errors that are localized for Firebase Sessions Library
+enum FirebaseSessionsError: Error {
+  /// Event sampled related error
+  case SessionSampledError
```

Comment:
Nit: "Sampled" by definition means to grab a subset from a set of data, so this could be confusing to think "error, because the session is being sent to backend?". I wonder if there's a better word for "this-session-is-being-dropped-error".

Reply:
This Error is just an error theme. Every error will have a module information and error description. This class defines just the module and not the actual message. So, this is just to reference that it is a "Sampling" related error. So, I would not worry about "event dropped message" in this context.